SConstruct

```diff
@@ -264,6 +264,7 @@ opts.Add(BoolVariable("builtin_brotli", "Use the built-in Brotli library", True)
 opts.Add(BoolVariable("builtin_certs", "Use the built-in SSL certificates bundles", True))
 opts.Add(BoolVariable("builtin_clipper2", "Use the built-in Clipper2 library", True))
 opts.Add(BoolVariable("builtin_embree", "Use the built-in Embree library", True))
+opts.Add(EnumVariable("builtin_embree_isa", "CPU instruction set extension used by Embree", "sse2", ("sse2", "sse42", "avx", "avx2", "avx512")))
```

Comment:
I'd add a warning about this, as the highest instruction set used becomes a hard requirement for running the compiled binary:

```suggestion
opts.Add(EnumVariable("builtin_embree_isa", "CPU instruction set extension used by Embree. Using an instruction set newer than SSE2 will prevent non-supported CPUs from running the compiled binary", "sse2", ("sse2", "sse42", "avx", "avx2", "avx512")))
```